src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs

```diff
@@ -0,0 +1,353 @@
+        /// </summary>
+        /// <exception cref="SigniFlow.CertifyCeremonyAPI.Client.ApiException">Thrown when fails to make API call</exception>
+        /// <param name="certifyingCeremonyRequest"></param>
+        /// <param name="operationIndex">Index associated with the operation.</param>
+        /// <returns>CertifyingCeremonyResponse</returns>
+        public CertifyingCeremonyResponse CertifyingCeremony(CertifyingCeremonyRequest certifyingCeremonyRequest, int operationIndex = 0)
+        {
+            SigniFlow.CertifyCeremonyAPI.Client.ApiResponse<CertifyingCeremonyResponse> localVarResponse = CertifyingCeremonyWithHttpInfo(certifyingCeremonyRequest);
+            return localVarResponse.Data;
+        }
+
+        /// <summary>
+        /// Certify a document 
+        /// </summary>
+        /// <exception cref="SigniFlow.CertifyCeremonyAPI.Client.ApiException">Thrown when fails to make API call</exception>
+        /// <param name="certifyingCeremonyRequest"></param>
+        /// <param name="operationIndex">Index associated with the operation.</param>
+        /// <returns>ApiResponse of CertifyingCeremonyResponse</returns>
+        public SigniFlow.CertifyCeremonyAPI.Client.ApiResponse<CertifyingCeremonyResponse> CertifyingCeremonyWithHttpInfo(CertifyingCeremonyRequest certifyingCeremonyRequest, int operationIndex = 0)
+        {
+            // verify the required parameter 'certifyingCeremonyRequest' is set
+            if (certifyingCeremonyRequest == null)
+            {
+                throw new SigniFlow.CertifyCeremonyAPI.Client.ApiException(400, "Missing required parameter 'certifyingCeremonyRequest' when calling CertifyingCeremonyApi->CertifyingCeremony");
+            }
+
+            SigniFlow.CertifyCeremonyAPI.Client.RequestOptions localVarRequestOptions = new SigniFlow.CertifyCeremonyAPI.Client.RequestOptions();
+
+            string[] _contentTypes = new string[] {
+                "application/json"
+            };
+
+            // to determine the Accept header
+            string[] _accepts = new string[] {
+                "application/json",
+                "application/xml"
+            };
+
+            var localVarContentType = SigniFlow.CertifyCeremonyAPI.Client.ClientUtils.SelectHeaderContentType(_contentTypes);
+            if (localVarContentType != null)
+            {
+                localVarRequestOptions.HeaderParameters.Add("Content-Type", localVarContentType);
+            }
+
+            var localVarAccept = SigniFlow.CertifyCeremonyAPI.Client.ClientUtils.SelectHeaderAccept(_accepts);
+            if (localVarAccept != null)
+            {
+                localVarRequestOptions.HeaderParameters.Add("Accept", localVarAccept);
+            }
+
+            localVarRequestOptions.Data = certifyingCeremonyRequest;
+
+            localVarRequestOptions.Operation = "CertifyingCeremonyApi.CertifyingCeremony";
+            localVarRequestOptions.OperationIndex = operationIndex;
+
+
```

Comment:
![RA3 - Spaaaaace](https://github.com/user-attachments/assets/81551109-887e-4b50-8973-7ae27989136d)

Reply:
This is in generated code